collect-tips.rst

```diff
@@ -0,0 +1,7 @@
+**************************
+Best Practices and Tips
+**************************
+
+- Specifying the GPS requirements can help users decide the accuracy of location they send and hence help save battery. For more information on enabling GPS services and its relation with battery life visit `this site <https://support.google.com/nexus/answer/3467281?hl=en>`_.
+
+- Encrypting the form ensures that data is safe even when sent over an HTTP connection and that it can be stored on a server that is not secure. The forms are not encrypted until they are finalized. However, the form data can be encrypted while they're in saved staten also by turning on Android device encryption. To encrypt your device, follow the instructions given `here <https://www.howtogeek.com/141953/how-to-encrypt-your-android-phone-and-why-you-might-want-to/>`_. 
```

Comment:
There are multiple ways that encryption interacts with ODK and I think it's important to spell all three out or it can confuse implementers.

1. You can encrypt the transition of data between device and server. This is what happens when you use HTTPS. It's a good idea to do and besides having to install an SSL certificate on your server (it comes automatically on App Engine) has no real downsides.
2. You can use public/private key encryption to encrypt finalized forms at rest. https://opendatakit.org/help/encrypted-forms spells this out. This has some downsides in that you encrypt only on finalizing and once you do, you can't edit. You also need ODK Briefcase to download and decrypt so you can't view data on the server.
3. You can encrypt everything on your Android device, which is what I think this tip was originally about. If your device supports this, it's definitely a good idea because it secures everything if your device gets lost/stolen.